spec/models/assessment_spec.rb

```diff
@@ -66,7 +66,7 @@
 
   context 'automatic calculation of total' do
     it 'calculates the total on save' do
-      ass = create :assessment, expenses: 102.33, fees: 44.86
+      create :assessment, expenses: 102.33, fees: 44.86
```

Comment:
hmm can't see how this is used. is this created assessment create in the claim?
`create :assessment, expenses: 102.33, fees: 44.86, claim: claim` for example